src/data/mongodb/interfaces/mongo-options.interface.ts

```diff
@@ -0,0 +1,4 @@
+export interface Options {
```

Comment:
Te recomiendo colocarle este nombre, no sabes si más adelante exista otra interfaz "Options"
```suggestion
export interface MongoOptions {
```